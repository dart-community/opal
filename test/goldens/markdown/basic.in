# Hello world

This is **bold** text.

This is _italic_ text.

This is `inline code`.

## Lists

- Item 1
- Item 2
- Item 3

1. First
2. Second
3. Third

## Code block

```dart
void main() {
  print('Hello world!');
}
```

## Links

Make sure to read the [Dart documentation](https://dart.dev).

## Blockquotes

> This is a blockquote
> that spans multiple lines.
