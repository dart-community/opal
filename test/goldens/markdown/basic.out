# Hello world
^^^^^^^^^^^^^
source-text-markup-markdown-tag-heading

This is **bold** text.
^
source-text-markup-markdown-unknown
 ^
 source-text-markup-markdown-unknown
  ^
  source-text-markup-markdown-unknown
   ^
   source-text-markup-markdown-unknown
    ^
    source-text-markup-markdown-whitespace
     ^
     source-text-markup-markdown-unknown
      ^
      source-text-markup-markdown-unknown
       ^
       source-text-markup-markdown-whitespace
        ^^
        source-text-markup-markdown-tag-strong-punctuation-strong-marker
          ^^^^
          source-text-markup-markdown-tag-strong-tag-strong
              ^^
              source-text-markup-markdown-tag-strong-punctuation-strong-marker
                ^
                source-text-markup-markdown-whitespace
                 ^
                 source-text-markup-markdown-unknown
                  ^
                  source-text-markup-markdown-unknown
                   ^
                   source-text-markup-markdown-unknown
                    ^
                    source-text-markup-markdown-unknown
                     ^
                     source-text-markup-markdown-unknown

This is _italic_ text.
^
source-text-markup-markdown-unknown
 ^
 source-text-markup-markdown-unknown
  ^
  source-text-markup-markdown-unknown
   ^
   source-text-markup-markdown-unknown
    ^
    source-text-markup-markdown-whitespace
     ^
     source-text-markup-markdown-unknown
      ^
      source-text-markup-markdown-unknown
       ^
       source-text-markup-markdown-whitespace
        ^
        source-text-markup-markdown-tag-emphasis-punctuation-emphasis-marker
         ^^^^^^
         source-text-markup-markdown-tag-emphasis-tag-emphasis
               ^
               source-text-markup-markdown-tag-emphasis-punctuation-emphasis-marker
                ^
                source-text-markup-markdown-whitespace
                 ^
                 source-text-markup-markdown-unknown
                  ^
                  source-text-markup-markdown-unknown
                   ^
                   source-text-markup-markdown-unknown
                    ^
                    source-text-markup-markdown-unknown
                     ^
                     source-text-markup-markdown-unknown

This is `inline code`.
^
source-text-markup-markdown-unknown
 ^
 source-text-markup-markdown-unknown
  ^
  source-text-markup-markdown-unknown
   ^
   source-text-markup-markdown-unknown
    ^
    source-text-markup-markdown-whitespace
     ^
     source-text-markup-markdown-unknown
      ^
      source-text-markup-markdown-unknown
       ^
       source-text-markup-markdown-whitespace
        ^
        source-text-markup-markdown-source-source-backtick
         ^^^^^^^^^^^
         source-text-markup-markdown-source-source-content
                    ^
                    source-text-markup-markdown-source-source-backtick
                     ^
                     source-text-markup-markdown-unknown

## Lists
^^^^^^^^
source-text-markup-markdown-tag-heading

- Item 1
^^
source-text-markup-markdown-punctuation-list-marker
  ^
  source-text-markup-markdown-unknown
   ^
   source-text-markup-markdown-unknown
    ^
    source-text-markup-markdown-unknown
     ^
     source-text-markup-markdown-unknown
      ^
      source-text-markup-markdown-whitespace
       ^
       source-text-markup-markdown-unknown
- Item 2
^^
source-text-markup-markdown-punctuation-list-marker
  ^
  source-text-markup-markdown-unknown
   ^
   source-text-markup-markdown-unknown
    ^
    source-text-markup-markdown-unknown
     ^
     source-text-markup-markdown-unknown
      ^
      source-text-markup-markdown-whitespace
       ^
       source-text-markup-markdown-unknown
- Item 3
^^
source-text-markup-markdown-punctuation-list-marker
  ^
  source-text-markup-markdown-unknown
   ^
   source-text-markup-markdown-unknown
    ^
    source-text-markup-markdown-unknown
     ^
     source-text-markup-markdown-unknown
      ^
      source-text-markup-markdown-whitespace
       ^
       source-text-markup-markdown-unknown

1. First
^^^
source-text-markup-markdown-punctuation-list-marker
   ^
   source-text-markup-markdown-unknown
    ^
    source-text-markup-markdown-unknown
     ^
     source-text-markup-markdown-unknown
      ^
      source-text-markup-markdown-unknown
       ^
       source-text-markup-markdown-unknown
2. Second
^^^
source-text-markup-markdown-punctuation-list-marker
   ^
   source-text-markup-markdown-unknown
    ^
    source-text-markup-markdown-unknown
     ^
     source-text-markup-markdown-unknown
      ^
      source-text-markup-markdown-unknown
       ^
       source-text-markup-markdown-unknown
        ^
        source-text-markup-markdown-unknown
3. Third
^^^
source-text-markup-markdown-punctuation-list-marker
   ^
   source-text-markup-markdown-unknown
    ^
    source-text-markup-markdown-unknown
     ^
     source-text-markup-markdown-unknown
      ^
      source-text-markup-markdown-unknown
       ^
       source-text-markup-markdown-unknown

## Code block
^^^^^^^^^^^^^
source-text-markup-markdown-tag-heading

```dart
^^^^^^^
source-text-markup-markdown-source-source-fence
void main() {
^^^^^^^^^^^^^
source-text-markup-markdown-source-source-content
  print('Hello world!');
^^^^^^^^^^^^^^^^^^^^^^^^
source-text-markup-markdown-source-source-content
}
^
source-text-markup-markdown-source-source-content
```
^^^
source-text-markup-markdown-source-source-fence

## Links
^^^^^^^^
source-text-markup-markdown-tag-heading

Make sure to read the [Dart documentation](https://dart.dev).
^
source-text-markup-markdown-unknown
 ^
 source-text-markup-markdown-unknown
  ^
  source-text-markup-markdown-unknown
   ^
   source-text-markup-markdown-unknown
    ^
    source-text-markup-markdown-whitespace
     ^
     source-text-markup-markdown-unknown
      ^
      source-text-markup-markdown-unknown
       ^
       source-text-markup-markdown-unknown
        ^
        source-text-markup-markdown-unknown
         ^
         source-text-markup-markdown-whitespace
          ^
          source-text-markup-markdown-unknown
           ^
           source-text-markup-markdown-unknown
            ^
            source-text-markup-markdown-whitespace
             ^
             source-text-markup-markdown-unknown
              ^
              source-text-markup-markdown-unknown
               ^
               source-text-markup-markdown-unknown
                ^
                source-text-markup-markdown-unknown
                 ^
                 source-text-markup-markdown-whitespace
                  ^
                  source-text-markup-markdown-unknown
                   ^
                   source-text-markup-markdown-unknown
                    ^
                    source-text-markup-markdown-unknown
                     ^
                     source-text-markup-markdown-whitespace
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                      source-text-markup-markdown-link
                                                            ^
                                                            source-text-markup-markdown-unknown

## Blockquotes
^^^^^^^^^^^^^^
source-text-markup-markdown-tag-heading

> This is a blockquote
^^^^^^^^^^^^^^^^^^^^^^
source-text-markup-markdown-punctuation-block-quote
> that spans multiple lines.
^^^^^^^^^^^^^^^^^^^^^^^^^^^^
source-text-markup-markdown-punctuation-block-quote
